### ISSUE TITLE
Proposal: Allow //example.com to be a valid url (no http(s): in front of it)

### ISSUE THREAD

**Issue by timothypratley** [jkk/verily#9]
Hi, thanks for making verily; I find it very useful :)

Would you accept a pull request to loosen the CLJS regex so that it will accept urls and web_urls without the scheme, such as
`//example.com` instead of requiring `http://example.com`?

It is often useful to leave the http(s) out so that the browser will use the same scheme as the current page when following links. I believe they are valid urls:
http://stackoverflow.com/questions/550038/is-it-valid-to-replace-http-with-in-a-script-src-http

Basically I propose to change the regex:
(re-find #"^[a-zA-Z]+://" %)
Such that it is optional to have the `http:` part at the front.

Please let me know what you think, I realize I can make a custom validator so this is not a blocker for me... I'm just raising it to see if this would be desirable.

**Comment by iku000888** [comment]
Hi TIm,
So I am asking jkk to give me commit rights to this repo as I started maintaining formative already, so I should be able to take PRs once that is done.

In terms of the suggestion, I would prefer a new relaxed validator with a new name such as 'relaxed-url' to avoid changing the existing semantics. What do you think?

**Comment by timothypratley** [comment]
Sounds good to me :)

https://github.com/jkk/verily/pull/11

I used the name `link-url` instead of `relaxed-url` because I think it conveys a little more clearly that it allows relative links. But I'm happy to switch it to `relaxed-url` if you think that is more appropriate.
